### DISCUSSION TITLE
Post install on AWS - unable to SSH

### DISCUSSION THREAD

**Discussion by eladrave** [Ysurac/openmptcprouter#3458] [Q&A]
I installed using the script on an AWS EC2 instance.
Rebooted after the install

Trying to ssh -p 65222 openmptcprouter@3.85.x.y
I get this message:
openmptcprouter@3.85.x.y: Permission denied (publickey).

On AWS, ports 65000 - 65535 on TCP and UDP are open

**Comment by Ysurac** [comment]
Are you sure "openmptcprouter" is the correct user ? It was working with this user before VPS script install ?

**Comment by mildly-longsword** [comment]
I just setup an AWS Lightsail vps earlier this week. After the OMR install I couldn't ssh in directly either, however using Putty and creating a key following the instructions [here](https://docs.aws.amazon.com/en_us/lightsail/latest/userguide/lightsail-how-to-set-up-putty-to-connect-using-ssh.html) allowed me to connect.
